.github/workflows/health_checks.yml

```diff
@@ -43,6 +43,37 @@ jobs:
       - uses: ./.github/actions/restore_build_cache
       - run: npm run set-script-shell
       - run: npm run test:coverage:threshold
+  test_scripts:
```

Comment:
I have separated these tests.
They are not very useful when working on `packages/*` and start growing.